src/libraries/System.Diagnostics.Tracing/tests/TrimmingTests/EventSourceManifestTest.cs

```diff
@@ -34,6 +35,10 @@ internal class EventSourceTest : EventSource
         int EventSourceTest_Method_7() => 5; 
     }
 
+    [UnconditionalSuppressMessage ("TrimAnalysis", "IL2111:RequiresDynamicallyAccessedMembers",
```

Comment:
Isn't this a test? Are we only getting warnings here because the SDK isn't suppressing these new codes?

Reply:
Thanks, I didn't realize we were suppressing warnings in all tests. Fixed.